web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -139,7 +139,9 @@ const sfc = {
       }
     });
   },
-
+  mounted() {
+    this.fetchBranchesAndTags();
```

Comment:
Is it right to do so? For the "release list" page, there are many branch selectors, each of them do a "fetch"?

Reply:
That's true.. Maybe only fetch branches?

Reply:
updated the fetch to be called only first time the list is shown